heatpump.yaml

```diff
@@ -2464,6 +2358,490 @@ switch:
+          if ((new_value) != id(unmasked_value_register_210)) {
+            // ESP_LOGD("unmasked_value_register_210", "Set option to off p_s_1_supports_room_temperature_sensor_ta 0x%x -> 0x%x", id(unmasked_value_register_210), new_value);
+            id(unmasked_value_register_210) = new_value;
+            esphome::modbus_controller::ModbusCommandItem set_payload_command = esphome::modbus_controller::ModbusCommandItem::create_write_single_command(${devicename}, 210, new_value);
+            ${devicename}->queue_command(set_payload_command);
+          }
+  # Register: 210, Bit: 5, default: false, TODO: verify default
+  # Midea, YORK: PUMPI silent mode, 1; valid, 0: invalid
+  - platform: template
+    name: "Parameter Setting 1 Supports T1 Sensor"
+    id: "${devicename}_parameter_setting_1_supports_t1_sensor"
+    icon: mdi:eye
+    restore_mode: DISABLED
+    entity_category: config
+    optimistic : true
+    lambda: "return (id(unmasked_value_register_210) & 0x20) == 0x20;"
+    on_turn_on:
+      - lambda: |-
+          uint16_t checked_bit = 0x20;
+          uint16_t new_value = id(unmasked_value_register_210);  // The original unmasked value
+          new_value &= ~checked_bit;                             // Clear and set
+          new_value += checked_bit;
+          if ((new_value) != id(unmasked_value_register_210)) {
+            // ESP_LOGI("unmasked_value_register_210", "Set option to on p_s_1_silent_mode_pump_i 0x%x -> 0x%x", id(unmasked_value_register_210), new_value);
+            id(unmasked_value_register_210) = new_value;
+            esphome::modbus_controller::ModbusCommandItem set_payload_command = esphome::modbus_controller::ModbusCommandItem::create_write_single_command(${devicename}, 210, new_value);
+            ${devicename}->queue_command(set_payload_command);
+          }
+    on_turn_off:
+      - lambda: |-
+          uint16_t checked_bit = 0x20;
+          uint16_t new_value = id(unmasked_value_register_210);  // The original unmasked value
+          new_value &= ~checked_bit;                             // Clear and set
+          if ((new_value) != id(unmasked_value_register_210)) {
+            // ESP_LOGI("unmasked_value_register_210", "Set option to off p_s_1_silent_mode_pump_i 0x%x -> 0x%x", id(unmasked_value_register_210), new_value);
+            id(unmasked_value_register_210) = new_value;
+            esphome::modbus_controller::ModbusCommandItem set_payload_command = esphome::modbus_controller::ModbusCommandItem::create_write_single_command(${devicename}, 210, new_value);
+            ${devicename}->queue_command(set_payload_command);
+          }
+  # Register: 210, Bit: 6, default: false, TODO: verify default
+  - platform: template
+    name: "Parameter Setting 1 T1S Heating High Low Temperature Settings"
+    id: "${devicename}_parameter_setting_1_t1s_heating_high_low_temperature_settings"
+    icon: mdi:eye
+    restore_mode: DISABLED
+    #entity_category: config
```

Comment:
Is there a specific reason to comment this out? If it is not needed, then please remove it

Reply:
entity_category -moves that to Configuration in homeassistant -so it is more intuitive ;)